<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Info</class>
 <widget class="QDialog" name="Info">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>485</width>
    <height>301</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>441</width>
     <height>271</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="3" column="0">
     <widget class="QLabel" name="Surname">
      <property name="text">
       <string>Surname:</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="Name">
      <property name="text">
       <string>Name:</string>
      </property>
     </widget>
    </item>
    <item row="7" column="0">
     <widget class="QLabel" name="eviction_date">
      <property name="text">
       <string>Eviction date:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="Room">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>Room:</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="Checkin_date">
      <property name="text">
       <string>Check-in date:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="F_name">
      <property name="text">
       <string>Family Name:</string>
      </property>
     </widget>
    </item>
    <item row="8" column="0" colspan="2">
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="1" column="2">
       <widget class="QPushButton" name="pushButton">
        <property name="text">
         <string>OK</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="Save">
        <property name="text">
         <string>Save</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="View">
        <property name="text">
         <string>View information</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QPushButton" name="pushButton_2">
        <property name="text">
         <string>Cancel</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="Birth_date">
      <property name="text">
       <string>Birth date:</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1" colspan="2">
     <widget class="QLineEdit" name="Name_push"/>
    </item>
    <item row="1" column="1" colspan="2">
     <widget class="QLineEdit" name="F_name_push"/>
    </item>
    <item row="0" column="1" colspan="2">
     <widget class="QSpinBox" name="Room_push"/>
    </item>
    <item row="6" column="1">
     <widget class="QDateEdit" name="Checkin_date_push"/>
    </item>
    <item row="7" column="1">
     <widget class="QDateEdit" name="Eviction_date_push"/>
    </item>
    <item row="5" column="1">
     <widget class="QDateEdit" name="Birth_date_push"/>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="Surname_push"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
